grafana-plugin/pkg/plugin/debug.go

```diff
@@ -7,6 +7,17 @@ import (
 	"net/http"
 )
 
+type OnCallDebugStats struct {
+	SettingsCallCount           int32 `json:"settingsCallCount"`
+	AllUsersCallCount           int32 `json:"allUsersCallCount"`
+	PermissionsCallCount        int32 `json:"permissionsCallCount"`
+	AllPermissionsCallCount     int32 `json:"allPermissionsCallCount"`
+	TeamForUserCallCount        int32 `json:"teamForUserCallCount"`
+	AllTeamsCallCount           int32 `json:"allTeamsCallCount"`
+	TeamMembersForTeamCallCount int32 `json:"teamMembersForTeamCallCount"`
+	CheckHealthCallCount        int32 `json:"checkHealthCallCount"`
```

Comment:
Curious, could we assume there is a health call per each plugin request? (trying to think how we map these counts to API calls)

Reply:
No we only do the check health call on status (initial load) and install since it is a waste for other calls since it will be contacting the engine anyways and should be already working.